storybooks/storybooks.github.io
# Add google webmaster verification page

Issue: #15

## What I did
Also, simplify build script by moving CNAME to pages.

We can’t do this for the google page because Gatsby converts x.html to
x/index.html

## How to test
```
yarn build
```
Check that CNAME and googleXXX.html are available in the generated `public` directory.
